week-1/errors/4.js

```diff
@@ -1,2 +1,2 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const twelveClockTime = "20:53";
+const twentyFourhourClockTime = "08:53";
```

Comment:
Wonderful work!

Just change twentyFourhourClockTime to twentyFourHourClockTime <-- This is known as camelCase

Reply:
well done!

Reply:
Thank you Marcus. Done!